.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ name: Continuous Integration
 on:
   push:
   pull_request:
-    branches: [ bleed ]
+    branches: [ bleed, 'prep-*' ]
```

Comment:
Why not
```suggestion
    branches: [ bleed, 'prep-**' ]
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Why do we need to match the `/` character?

Reply:
I don't see why we need to exclude it

Reply:
Prep branches don't or shouldn't contain that character, so I don't see the benefit of using the less obvious operator.

Reply:
If it comes back to bite us in the arse I'll blame you \:P